iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/shared_memory_object.hpp

```diff
@@ -49,7 +49,7 @@ class SharedMemoryObject
   public:
     using Builder = SharedMemoryObjectBuilder;
 
-    static constexpr void* NO_ADDRESS_HINT = nullptr;
+    static constexpr const void* const NO_ADDRESS_HINT = nullptr;
```

Comment:
Isn't the address hint deprecated for a long time? Couldn't it be removed?

Reply:
No, not at all the posix wrapper still provides this feature and there is no need to remove it.
posh can now handle any arbitrary address but since this is a low level building block I do not see the need for removing and already working and tested feature.